properties/size/breakpoint.json

```diff
@@ -0,0 +1,22 @@
+{
+  "size": {
+    "breakpoint": {
+      "sm": {
```

Comment:
update breakpoint names to be more descriptive (and different from our spacing names)

e.g. 'phone, tablet, desktop, hd'